pkgs/development/compilers/llvm/common/llvm/default.nix

```diff
@@ -381,6 +393,8 @@ stdenv.mkDerivation (rec {
         nativeInstallFlags
       ])
     )
+  ] ++ optionals (!enableTerminfo) [
+    "-DLLVM_ENABLE_TERMINFO=OFF"
```

Comment:
Not opposed to this, just curious: what required this?

Reply:
By plunging into the icy depths of the commit history from several branches ago, I was able to determine this was added while I was trying to get a bunch of things to compile cross-static for the bootstrap files. Sounds like I couldn't get a staticlib out of terminfo.

Reply:
Can you mention this in the commit message?

Reply:
Done. I was also able to clarify some of the other parts of the commit message.

Reply:
...I'm very curious what your perception of the need to go to staging is, if not "rebuild-darwin-stdenv". But yes.

Reply:
Oh I didn't see that, haha. But why is it the case? The code looks pretty innocent.

Reply:
https://github.com/NixOS/nixpkgs/pull/311777/files#diff-ce640f96a649fa4a9dd17247b89883c49921ef7e013fb8ca31bafc65e03d298fL145 adds to a comment in a darwin-gated shell script

Reply:
Oh. Let's just add a comment outside the script in Nix code then. And next `staging` we should move the rest of the comment outside too so we don't get this problem again.

Reply:
Why would it even matter if it went through staging?  There's no binary cache for FreeBSD anyway.

Reply:
@alyssais Good point. But I guess it's still nice to quickly test that cross still works etc. too so that can help.